### ISSUE TITLE
opencvsharp4.runtime for android-arm32 and android-x32

### ISSUE THREAD

**Issue by WangWeijie1998** [sdcb/opencvsharp-mini-runtime#4]
Hi,thanks for you work! I try make project with opencvsharp+android11-armeadi-v7a+.net maui, opencvsharp can't use. Your android runtime package is contains only arm64 and x64, if you figure out that problem with a runtime package which contains other cup architectures it may work.

**Comment by sdcb** [comment]
No, I'm not interested in arm32/x86 in android, I think no one use it
